dev/com.ibm.websphere.appserver.features/visibility/public/appSecurity-4.0/resources/l10n/io.openliberty.appSecurity-4.0.properties

```diff
@@ -0,0 +1,18 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables support for securing the server runtime environment and applications \
+ using Security-1.0 as defined in JSR-375.
```

Comment:
Clearly this message is wrong.

Reply:
removed the JSR reference and re-wrote as: "... using Jakarta Security-2.0."

Reply:
Still appears to say `Security-1.0 as defined in JSR-375`

Reply:
He hasn't pushed the update yet.